renku/errors.py

```diff
@@ -207,6 +207,36 @@ def __init__(self, repo, inputs):
         super(OutputsNotFound, self).__init__(msg)
 
 
+class ResourceNotFound(RenkuException, click.ClickException):
+    """Raise when resource is not found."""
+
+    def __init__(self, resource_type):
+        """Build a custom message."""
+        msg = 'Specified {0} resource could not be found.'.format(
+            resource_type
+        )
+
+        super(ResourceNotFound, self).__init__(msg)
```

Comment:
you can use `super()` since we support only Python 3.

```suggestion
        super().__init__(msg)
```